### ISSUE TITLE
Expose accessors so they can be composed

### ISSUE THREAD

**Issue by satazor** [evanshortiss/env-var#121]
**Is your feature request related to a problem? Please describe.**
I would like to access the internal accessors, so that I can create new ones, while using the built-in accessors internally.

**Describe the solution you'd like**
Exposing the accessors, like `env.accessors`.

**Describe alternatives you've considered**
At the moment, I'm importing `lib/accessors/xxx`, which is far from ideal, as they are not part of the public API.

**Additional context**

Example:

```js
        asIntRange: (value, range) => {
            value = asInt(value);

            if (value < range[0] || value >= range[1]) {
                throw new Error(`to be within range [${range[0]},${range[1]}]`);
            }

            return value;
        },
```

where `asInt` is the `lib/accessors/int.js`.

**Comment by evanshortiss** [comment]
@satazor I just want to clarify. Are you trying to use the internal accessors in one you plan to pass to [extraAccessors](https://github.com/evanshortiss/env-var#extraaccessors)?

**Comment by satazor** [comment]
@evanshortiss correct

**Comment by evanshortiss** [comment]
@satazor gotcha, seems reasonable. Would you like to PR this?

**Comment by satazor** [comment]
I will put this into my TODO list, however don't wait on me and if anyone want's to tackle this please go ahead.

**Comment by evanshortiss** [comment]
@satazor this will be available in 6.2.0
